Mathlib/Data/Set/Lattice.lean

```diff
@@ -1872,6 +1872,11 @@ theorem iUnion_prod {ι ι' α β} (s : ι → Set α) (t : ι' → Set β) :
   simp
 #align set.Union_prod Set.iUnion_prod
 
+/-- Analogue of `iSup.prod` for sets. -/
+lemma Set.iUnion_prod' (f : β × γ → Set α) : ⋃ x : β × γ, f x = ⋃ (i : β) (j : γ), (f ⟨i, j⟩) := by
+  symm
+  simp only [iUnion, iSup_eq_iUnion, iSup_prod]
```

Comment:
```suggestion
lemma Set.iUnion_prod' (f : β × γ → Set α) : ⋃ x : β × γ, f x = ⋃ (i : β) (j : γ), f (i, j) := by
  simp only [iUnion, iSup_eq_iUnion, iSup_prod]
```
Interestingly, `⋃ (i) (j)` also works but `⋃ (i j)` and `⋃ i j` don't. I recall they all work in Lean 3 ...